<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <node name="razor_imu" pkg="rr_platform" type="razor_imu" output="screen">
        <param name="serial_port" value="/dev/razor_imu"/>
    </node>
</launch>